tests/conftest.py

```diff
@@ -344,6 +375,13 @@ def reload_modules(request: pytest.FixtureRequest):
             importlib.reload(mod)
 
 
+@pytest.fixture(scope="function")
+def manual_seed():
+    from morpheus.utils import seed as seed_utils
+    seed_utils.manual_seed(42)
+    yield
+
+
```

Comment:
Nice function. Couple of things:

1. Needs a docstring to explain what its being used for
2. This could also return a function to allow users to call it within the test. Like `yield lambda _: seed_utils.manual_seed(42)`